tests/Panel/Areas/InstallationTest.php

```diff
@@ -21,20 +21,18 @@ public function testInstallation()
 		$this->assertSame('Installation', $view['title']);
 		$this->assertSame('k-installation-view', $view['component']);
 
-		// cli is not a valid server, therefor installation fails
 		$this->assertFalse($view['props']['isInstallable']);
 		$this->assertFalse($view['props']['isInstalled']);
-		$this->assertFalse($view['props']['isOk']);
+		$this->assertTrue($view['props']['isOk']);
```

Comment:
With this change, the test does not actually test anymore whether the `isOk` prop switches to `false` on issues. We could instead e.g. make the `content` folder non-writable in the test.

Reply:
I don't think the test was really meant to test the false `isOk` – I think that was just an unavoidable circumstance back then when testing in CLI would not allow the server. But if I look at the rest of the test, I think this is meant to test a positive installation state.

We are testing whether `isOk` works or not in its regular tests in `SystemTest`. If we want to add more test cases there, we can, but I would keep this separate from this PR.

Reply:
I think we should absolutely have a test for that. Otherwise we could easily break the installation test by accident and we would only realize that after having debugged support requests for ages. IMO it's a regression to remove the test in this PR. A test for the "normal sunshine case" is not useful as it only provides code coverage without actually testing the behavior.

But it's fine for me to merge it first if you add a test separately, directly to the develop branch without another PR would be fine as well.

Reply:
But this test isn't testing `$system->isOk()`, it's testing that the installation view receives props. Testing `$system->isOk()` is important, I agree, but it should take place in `SystemTest`. I think we should avoid this kind of cross-testing where we can and rather have actual tests for these parts.

Reply:
The only change that could make sense here to really test the installation props functionality is to write

```php
$this->assertSame($system->isOk(), $view['props']['isOk']);
```

It seems a bit redundant, but that's the only thing we should test in this test case: is the prop providing the state our system checks return.

Reply:
Ah, right. I missed that this is a test for the area. I agree.